### ISSUE TITLE
Add support for YAML

### ISSUE THREAD

**Issue by engsbk** [atom-ide-community/atom-script#1227]

**Comment by rgbkrk** [comment]
What would "running" YAML do? Validate the structure?

Curiosity - is this for an Ansible playbook?

**Comment by engsbk** [comment]
Not really. I was trying to test a crawler code for youtube and the file is .yaml
I’m using Atom to run the file.


Thank you for taking the time to read this,
Shaikhah B. Alkhadhr

On Jan 25, 2017, at 10:33 PM, Kyle Kelley <notifications@github.com<mailto:notifications@github.com>> wrote:


What would "running" YAML do? Validate the structure?

Curiosity - is this for an Ansible playbook?

**Comment by rgbkrk** [comment]
Script operates by invoking some interpreter (runner) or compiler for standard languages. I'm curious if there's a way for you to use the script "run with options" command.

**Comment by engsbk** [comment]
I did actually. Cmd+shift+I then I specified yaml as the command, but got the same result.

Thanks for taking the time to read this,
Shaikhah B. Alkhadhr

On Jan 26, 2017, at 1:36 AM, Kyle Kelley <notifications@github.com<mailto:notifications@github.com>> wrote:


Script operates by invoking some interpreter (runner) or compiler for standard languages. I'm curious if there's a way for you to use the script "run with options" command.

-
